### DISCUSSION TITLE
Omada Open API Support

### DISCUSSION THREAD

**Discussion by codersaur** [charlie-haley/omada_exporter#80] [General]
I see in v5.12 on the Omada Controller Open API is now supported.

_Open API: Omada Controller now supports Open API integration. You can access it through Global view > Settings > Platform Integration, which allows you to utilize the REST API of most Controller services. This feature enables you to develop custom applications, embed APIs, or combine your own applications._

Is this relevant for omada_exporter, and if so, is there a plan to migrate to using it?

**Comment by charlie-haley** [comment]
This should make things a lot easier actually, I'll convert this to an issue as it's definitely something that should be implemented, thanks @codersaur
